docs/src/main/asciidoc/security-customization.adoc

```diff
@@ -620,6 +620,62 @@ are using an executor that is capable of propagating the identity (e.g. no `Comp
 to make sure that Quarkus can propagate it. For more information see the
 xref:context-propagation.adoc[Context Propagation Guide].
 
+[[observe-security-events]]
+== Observe security events
+
+Quarkus beans fire various security events that can be consumed by xref:cdi.adoc#events-and-observers[CDI observers].
```

Comment:
> I was not sure about Quarkus beans fire since they consume events if I understand it correctly

Security beans fire events and CDI observers consume them, IMO it is not incorrect, but as said, I like your version.